### PR TITLE
added recipes for openfile

### PR DESCRIPTION
**Pull request by monaldoj**
Added recipes for new python package:
 * openfile

These were built using `conda skeleton pypi`

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -1,83 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (xhochy) [inline recipes/example/meta.yaml]: Please keep the example

File: recipes/openfile/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ea4e8f19112049f578ec161b12f70c1113144988c0a488ab173a138b50f37479
+
+build:
+  number: 0
```
Comment (xhochy) [inline recipes/openfile/meta.yaml]:
```suggestion
  number: 0
  noarch: python
```

File: recipes/openfile/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  build:
+    - python 3.6
```
Comment (xhochy) [inline recipes/openfile/meta.yaml]: Why is the Python fixed? The package looks like it is support 3.x

Comment (xhochy) [reply recipes/openfile/meta.yaml]: The `build:` section should be empty, this is normally only needed for compilers.

File: recipes/openfile/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  imports:
+    - openfile
+
+about:
+  home: https://github.com/luismsgomes/openfile
+  license: MIT
+  license_family: MIT
+  license_file: 
+  summary: A convenience function that delegates to the appropriate standard library function.
+  doc_url:
```
Comment (xhochy) [inline recipes/openfile/meta.yaml]: Omit empty entries

**Comment by monaldoj** [comment]
@chrisburr i think you merged this just before i committed my changes to address @xhochy's suggestions. (they were essentially the same changes i made on `toolwrapper`)

**Pull request merged**